### PR TITLE
269 test getBaseAccount, getWalletAccounts

### PR DESCRIPTION
**Pull request by julienbrs**
testing getBaseAccount and setBaseAccount

### CODE REVIEW

File: test/web3/walletClient.spec.ts [hunk 1]
```diff
@@ -83,4 +91,52 @@ describe.skip('WalletClient', () => {
       expect(baseAccount?.address).toEqual(secondAccount.address);
     });
   });
+
+  describe('getBaseAccount', () => {
+    test('should return the base account', async () => {
+      const fetchedBaseAccount = await web3Client.wallet().getBaseAccount();
+      expect(fetchedBaseAccount).not.toBeNull();
+      console.log('fetchedBaseAccount', fetchedBaseAccount);
```
Comment (gregLibert) [inline test/web3/walletClient.spec.ts:99]: I guess it's for debug. can you remove it ?

File: test/web3/walletClient.spec.ts [hunk 2]
```diff
@@ -83,4 +91,52 @@ describe.skip('WalletClient', () => {
...
+    test('should return all accounts in the wallet', async () => {
+      const accounts = [
+        await WalletClient.walletGenerateNewAccount(),
+        await WalletClient.walletGenerateNewAccount(),
+        await WalletClient.walletGenerateNewAccount(),
+      ];
+
+      await web3Client.wallet().addAccountsToWallet(accounts);
+      const walletAccounts = await web3Client.wallet().getWalletAccounts();
+      expect(walletAccounts.length).toBe(4);
```
Comment (gregLibert) [inline test/web3/walletClient.spec.ts:120]: Why is it 4 and not 3 ?

**Review by gregLibert — CHANGES_REQUESTED** [review]

File: test/web3/walletClient.spec.ts (hunk 2 above)
Comment (julienbrs) [reply test/web3/walletClient.spec.ts:120]: there also is the baseAccount

Comment (gregLibert) [reply test/web3/walletClient.spec.ts:119]: Can you add a comment to explain that for the next dev who check the unit test ?

**Review by gregLibert — CHANGES_REQUESTED** [review]

**Pull request merged**